engine/tests/phpunit/unit/Elgg/Navigation/MenuRenderingTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Elgg\Navigation;
+
+use Elgg\IntegratedUnitTestCase;
+use Elgg\UnitTestCase;
+
+/**
+ * The purpose of these tests is to catch validation problems
+ * in menu hooks
+ *
+ * @group Navigation
+ */
+class MenuRenderingTest extends IntegratedUnitTestCase {
+
+	public function up() {
+		$this->user = $this->createUser();
+		elgg_set_page_owner_guid($this->user->guid);
+		$this->object = $this->createObject();
+		$this->group = $this->createGroup();
+
+		elgg_trigger_event('init', 'system');
+	}
+
+	public function down() {
+

```

Comment:
why not cleanup what you create in up()?

Reply:
Parent class cleans up db query specs